src/converter-gen/mvn-converters-xpl.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+# shellcheck source=../common/subcommand_common.bash
+source "$SCRIPT_DIR/../common/subcommand_common.bash"
+
+usage() {
+    cat <<EOF
+Usage: ${BASE_COMMAND:-$(basename "${BASH_SOURCE[0]}")} METASCHEMA_XML OUTPUT_DIR SCHEMA_NAME [ADDITIONAL_ARGS]
+
+Produces XML-to-JSON and JSON-to-XML conversion transformations (XSLTs) from Metaschema XML source, using XML Calabash invoked from Maven.
+Please install Maven first.
+
+Additional arguments for XML Calabash should be specified in the `key=value` format.
+EOF
+}
+
+if ! [ -x "$(command -v mvn)" ]; then
+  echo 'Error: Maven (mvn) is not in the PATH, is it installed?' >&2
+  exit 1
+fi
+
+[[ -z "${1-}" ]] && { echo "Error: METASCHEMA_XML not specified"; usage; exit 1; }
+METASCHEMA_XML=$1
+[[ -z "${2-}" ]] && { echo "Error: OUTPUT_DIR not specified"; usage; exit 1; }
+OUTPUT_DIR=$2
+[[ -z "${3-}" ]] && { echo "Error: SCHEMA_NAME not specified"; usage; exit 1; }
+SCHEMA_NAME=$3
+
+ADDITIONAL_ARGS=$(shift 3; echo "${*// /\\ }")
+
+PIPELINE="${SCRIPT_DIR}/METASCHEMA-ALL-SCHEMAS.xpl"
+
+XMLTOJSON_CONVERTER_FILE="${OUTPUT_DIR}/${SCHEMA_NAME}-xml-to-json.xsl"
+
+JSONTOXML_CONVERTER_FILE="${OUTPUT_DIR}/${SCHEMA_NAME}-json-to-xml.xsl"
+
+CALABASH_ARGS="-iMETASCHEMA=\"$METASCHEMA_XML\" \
+               -oINT_0_echo-input=/dev/null \
+               -oINT_1_composed-metaschema=/dev/null \
+               -oINT_2_initial-model-map=/dev/null \
+               -oINT_3_unfolded-model-ma=/dev/null \p
```

Comment:
Blocking: I know this came up in pairing on another PR or maybe it was this script, but we need to remove the errant `p` after this line escape.

```suggestion
               -oINT_3_unfolded-model-ma=/dev/null \
```

Reply:
Thought we fixed this but oh well, fixing.